docusaurus.config.js

```diff
@@ -1,97 +1,99 @@
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: "WeatherXM Docs",
-  tagline: "WeatherXM Documentation Page",
-  url: "https://docs.weatherxm.com",
-  baseUrl: "/",
-  onBrokenLinks: "throw",
-  onBrokenMarkdownLinks: "warn",
-  favicon: "img/favicon/favicon.ico",
-  organizationName: "WeatherXM", // Usually your GitHub org/user name.
-  projectName: "docs", // Usually your repo name.
-  deploymentBranch: 'gh-pages',
-  trailingSlash: false,
+    title: "WeatherXM Docs",
+    tagline: "WeatherXM Documentation Page",
+    url: "https://docs.weatherxm.com",
+    baseUrl: "/",
+    onBrokenLinks: "throw",
+    onBrokenMarkdownLinks: "warn",
+    favicon: "img/favicon/favicon.ico",
+    organizationName: "WeatherXM", // Usually your GitHub org/user name.
+    projectName: "docs", // Usually your repo name.
+    deploymentBranch: "gh-pages",
+    trailingSlash: false,
 
-  presets: [
-    'docusaurus-plugin-hotjar'
-  ],
+    scripts: [{
+        src: "/static/js/loadtag.js",
+        async: false,
+    }, ],
 
-  presets: [
-    [
-      "@docusaurus/preset-classic",
-      {
-        gtag: {
-          trackingID: 'G-E38CY3JRM3'
-        },
-        docs: {
-          sidebarPath: require.resolve("./sidebars.js"),
-          routeBasePath: "/",
-          // Please change this to your repo.
-          editUrl: "https://github.com/WeatherXM/docs/blob/main",
-        },
-        theme: {
-          customCss: require.resolve("./src/css/custom.css"),
-        },
-      },
+    presets: ["docusaurus-plugin-hotjar"],
```

Comment:
This may be the reason your script was not included when using the "docusaurus way". You have added a second `presets` attributes, which is being replaced by the next one. JavaScript objects don't support having twice the same property. This is, by the way, something a proper IDE would catch, like WebStorm ;-)